suitcase/hdf5.py

```diff
@@ -96,25 +96,23 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
                     data = np.array(data)
-                    try:
+                    if data.dtype.kind in ['S', 'U']:
```

Comment:
Fair enough, @tacaswell, but how can we move forward with this PR? I would suggest @licode check the self-reported dtype and then, if an error is caught, try inferring the actual datatype instead. Thoughts?